contrib/syslog-debun

```diff
@@ -345,18 +376,28 @@ acquire_general_info () {
 acquire_syslog_config () {
 	echo "Copy configs from $confdir"
 	cd $confdir
-	mkdir  ${tmpdir}/config
-	find . | cpio -pd ${tmpdir}/config
+	mkdir ${tmpdir}/config
+	find . > ${tmpdir}/syslog.etc.files
+	grep '\.key$' ${tmpdir}/syslog.etc.files > ${tmpdir}/syslog.etc.files.removed
+	grep -v '\.key$' ${tmpdir}/syslog.etc.files | \
+		while read FILE; do \
+			if ${grepq} "PRIVATE KEY-----" "${FILE}" 2>/dev/null; then
+				echo "${FILE}" >> ${tmpdir}/syslog.etc.files.removed
+			else
+				echo "${FILE}"
+			fi
```

Comment:
For now, I don't plan on implementing this in this current iteration, as it would take too much time.
I will put this on the wishlist for the next version.
I have added some extra code to generate the list of removed files. In later versions this feature could work with that list.